gqlalchemy/transformations.py

```diff
@@ -32,14 +32,14 @@ class NetworkXGraphConstants:
     ID = "id"
 
 
-def nx_to_cypher(graph: nx.Graph, create_index=False) -> Iterator[str]:
+def nx_to_cypher(
+    graph: nx.Graph, create_index=False, config: NetworkXCypherConfig = NetworkXCypherConfig()
```

Comment:
Not sure how such created default named parameters would act. Not sure what is the practice if the named parameter is an object.

Reply:
It's best to avoid default mutable arguments, I cannot see the problem here but it's generally been known to cause issues with collections (a good article that explains what could happen https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/).